src/background.js

```diff
@@ -199,25 +201,28 @@ async function clearFacebookCookies () {
 
 async function setupContainer () {
   // Use existing Facebook container, or create one
-  const contexts = await browser.contextualIdentities.query({name: FACEBOOK_CONTAINER_NAME});
+
+  const info = await browser.runtime.getBrowserInfo();
+  if (parseInt(info.version) < 67) {
+    FACEBOOK_CONTAINER_DETAILS.color = "blue";
+    FACEBOOK_CONTAINER_DETAILS.icon = "briefcase";
```

Comment:
I'm not super-pleased with changing properties of a `const` object, but this still feels cleaner. 😐

Reply:
Well, assigning stuff to `const` and modifying its content anway seems to be best-practice in the es6-world, so imho that's fine.